packages/account-v2/src/components/SignupWizard/index.tsx

```diff
@@ -1,41 +1,69 @@
-import React, { useEffect } from 'react';
+import React, { useCallback, useEffect } from 'react';
 import ReactModal from 'react-modal';
 import { StandaloneXmarkBoldIcon } from '@deriv/quill-icons';
-import { useSignupWizardContext } from '../../context/SignupWizardContext';
+import { ACTION_TYPES, useSignupWizardContext } from '../../context/SignupWizardContext';
+import { CUSTOM_STYLES } from '../../helpers/signupModalHelpers';
+import { stepProgress } from '../../mocks/form-progress.mock';
 import CurrencySelector from '../../pages/CurrencySelector';
-import { CUSTOM_STYLES } from './helpers';
+import { FormProgress } from '../form-progress';
 import './index.scss';
 
+/**
+ * @name WizardScreens
+ * @description The WizardScreens component is used to render the screens in the SignupWizard component based on the active step from `SignupWizardContext`.
+ * @example
+ * return (
+ *   <WizardScreens />
+ * );
+ */
 const WizardScreens = () => {
     const { currentStep } = useSignupWizardContext();
 
     // eslint-disable-next-line sonarjs/no-small-switch
```

Comment:
do we still need this ?

Reply:
I guess not . will remove it. Thanks for noticing @amina-deriv